katalog/harbor/trivy/kustomization.yaml

```diff
@@ -19,7 +19,7 @@ commonLabels:
 
 images:
   - name: goharbor/trivy-adapter-photon
-    newName: registry.sighup.io/fury/goharbor/trivy-adapter-photon
+    newName: quay.io/sighup/trivy-adapter-photon-offline
```

Comment:
This should use our own registry using the container image sync repo (the repo is updated once a day)